src/bit-u.md

```diff
@@ -308,7 +308,7 @@ a.quote-link:hover{
   <div class="academy-section academy-logos">
     <a href="./learn-advanced-javascript.html"><img class="academy-course-logo" src="./static/img/javascript.png"></a>
     <a href="./learn-angular.html"><img class="academy-course-logo" src="./static/img/angular.png"></a>
-    <a href="./learn-dom-jquery.html"><img class="academy-course-logo" src="./static/img/jquery-logo.png"></a>
+    <a href="./learn-react.html"><img class="academy-course-logo" src="./static/img/react.png"></a>
```

Comment:
I replaced the jquery item in the "main menu" with React. If there is a reason not to make React this prominent then deny this PR, but it does seem strange. @tehfedaykin @christopherjbaker

Reply:
Where is this page rendered? I can’t find it.

Reply:
@chasenlehara These are the logos right below the banner on the [homepage](https://www.bitovi.com/academy/). 
<img width="1499" alt="image" src="https://github.com/bitovi/academy/assets/980338/8e0a0bbf-a072-4d17-b69b-aa74765fbe2e">

Reply:
Ah ok, thank you!

@rlmcneary2 If you make any other changes to this file, it’d be great to get some `alt` attributes on those images.